### PR TITLE
Added Service Fabric Sample

### PR DESCRIPTION
**Pull request by Robbie-Microsoft**
This sample is almost identical to the Azure VM sample. Only the README really needs to be reviewed.

### CODE REVIEW

File: samples/msal-node-samples/Managed-Identity/ServiceFabric/HttpClient.ts [hunk 1]
```diff
@@ -0,0 +1,36 @@
+import https from "https";
```
Comment (bgavrilMS) [inline samples/msal-node-samples/Managed-Identity/ServiceFabric/HttpClient.ts]: I find it a bit strange that this file is named "HttpClient" as that has a lot of different meaning. Maybe call it "KeyVaultAccessor" or "DownstreamApi" ?

**Review by bgavrilMS — APPROVED** [review]

File: samples/msal-node-samples/Managed-Identity/ServiceFabric/index.ts [hunk 2]
```diff
@@ -0,0 +1,98 @@
+import { LoggerOptions } from "@azure/msal-common";
```
Comment (bgavrilMS) [inline samples/msal-node-samples/Managed-Identity/ServiceFabric/index.ts:1]: Is there anything different in this sample vs the IMDS sample? (except the Readme?). Should these be consolidated?

Comment (Robbie-Microsoft) [reply samples/msal-node-samples/Managed-Identity/ServiceFabric/index.ts:1]: The use of expressjs (and the README) are the only differences. Express is needed to serve the application to the Service Fabric's website.

Comment (neha-bhargava) [reply samples/msal-node-samples/Managed-Identity/ServiceFabric/index.ts:1]: The IMDS sample is a standalone console app vs this one is an app/ website that can be deployed to an app service or service fabric

File: samples/msal-node-samples/Managed-Identity/Imds/README.md [hunk 3]
```diff
@@ -67,5 +67,3 @@ An npm script has been configured in package.json, which will run both of the ab
 ```console
     npm run start:app
 ```
-
-A token will be returned from the system assigned managed identity application as well as the user assigned managed identity application, and they will both be immediately displayed in the terminal.
```
Comment (neha-bhargava) [inline samples/msal-node-samples/Managed-Identity/Imds/README.md:71]: 👍

File: samples/msal-node-samples/Managed-Identity/ServiceFabric/README.md [hunk 4]
```diff
@@ -0,0 +1,80 @@
...
+
+After the Service Fabric cluster has been created via the above guide, managed identity must be enabled for it. To do so, navigate to "Virtual machine scale sets" in the Azure portal. Click on the virtual machine scale set associated with the service fabric cluster that was just created. In the left sidebar, navigate to "Security > Identity". From here, system-assigned managed identity can be enabled and user-assigned managed identities can be created.
+
+## Key Vault Setup
+
+Follow the tutorial to provision the resources necessary for this sample: [Tutorial: Use a Windows VM system-assigned managed identity to access Azure Key Vault](https://learn.microsoft.com/en-us/entra/identity/managed-identities-azure-resources/tutorial-windows-vm-access-nonaad).
+
+### Note
+
+The above guide was created for Azure VMs, but is applicable for Service Fabric clusters too. In the "Select Principle" field, choose the name of the Virtual machine scale set associated with the Service Fabric cluster.
```
Comment (neha-bhargava) [inline samples/msal-node-samples/Managed-Identity/ServiceFabric/README.md]:
```suggestion
The above guide was created for Azure VMs, but is applicable for Service Fabric clusters too. In the "Select Principal" field, choose the name of the Virtual machine scale set associated with the Service Fabric cluster.
```

File: samples/msal-node-samples/Managed-Identity/ServiceFabric/README.md [hunk 5]
```diff
@@ -0,0 +1,80 @@
+# Managed Identity for Azure Service Fabric Cluster Sample
```
Comment (neha-bhargava) [inline samples/msal-node-samples/Managed-Identity/ServiceFabric/README.md:1]: Very clear steps for service fabric sample. So many learnings here :)

**Review by neha-bhargava — APPROVED** [review]

**Pull request merged**